hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/helpers/OmGetKey.java

```diff
@@ -0,0 +1,175 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ *  with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package org.apache.hadoop.ozone.om.helpers;
+
+import org.apache.hadoop.ozone.om.OMMetadataManager;
+import org.apache.hadoop.ozone.om.request.file.OMFileRequest;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Iterator;
+
+/**
+ * This class help to get metadata keys.
+ */
+public final class OmGetKey {
+
+  private String volumeName;
+  private String bucketName;
+  private String keyName;
+  private OMMetadataManager omMetadataManager;
+  private long clientID;
```

Comment:
How about we remove `clientID` from the helper class? Instead, we can pass the `clientID` in the `getOpenFileName(long clientID)`.

This makes the helper class reusable for getting different FSO-related DB keys, for example for FSO multipart open key (`OMMetadataManager#getMultipartKey`), which has the DB key format `/{volumeId}/{bucketId}/{parentId}/{fileName}/{uploadId}`, we can pass `uploadId` instead.

To see what I meant, you can refer to https://github.com/ivandika3/ozone/commit/4803bb27faa4b7790763d9952520fa62fd057d39#diff-fae9ce494b51c92df1ea1465582a7ce2ea29afaf3aa06008ce5f0a7b41a17cbcR140-R153